### PR TITLE
[Enhancement](full compaction) Add run status support for full compaction

### PR DESCRIPTION
**Pull request by Yukang-Lian**
The usage is `curl http://{ip}:{host}/api/compaction/run_status?tablet_id={tablet_id}`
e.g. `curl http://127.0.0.1:8040/api/compaction/run_status?tablet_id=10084`

If full compaction is running, the output will be
```
{
"status" : "Success",
"run_status" : true,
"msg" : "compaction task for this tablet is running",
"tablet_id" : 10084,
"compact_type" : "full"
}
```
else the ouput will be
```
{
"status" : "Success",
"run_status" : false,
"msg" : "compaction task for this tablet is not running",
"tablet_id" : 10084,
"compact_type" : "full"
}
```

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by carlvinhust2012** [comment]
LGTM

**Comment by Yukang-Lian** [comment]
run buildall

**Review by ByteYue — APPROVED** [review]

File: be/src/olap/tablet.h [hunk 1]
```diff
@@ -459,6 +459,10 @@ class Tablet final : public BaseTablet {
     void set_alter_failed(bool alter_failed) { _alter_failed = alter_failed; }
     bool is_alter_failed() { return _alter_failed; }
 
+    void set_is_full_compaction_running(bool is_full_compaction_running) {
+        _is_full_compaction_running = is_full_compaction_running;
+    }
+    inline bool is_full_compaction_running() const { return _is_full_compaction_running; }
```
Comment (gavinchou) [inline be/src/olap/tablet.h:465]: Do we need to make it thread-safe?

**Comment by Yukang-Lian** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**